### PR TITLE
LPS-130988 Replace completed query param to processStatuses

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:stop

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -939,6 +939,13 @@ paths:
                   schema:
                       format: date-time
                       type: string
+                - in: query
+                  name: processStatuses
+                  schema:
+                      items:
+                          enum: [ Completed, Pending ]
+                          type: string
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/rest-openapi.yaml:947]: @MarinhoFeliphe remove spaces `[ Completed, Pending ]` -> `[Completed, Pending]`. please

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
Just started reviewing :)

**Comment by rafaprax** [comment]
ci:stop

**Pull request closed without merging**